src/service/order.service.ts

```diff
@@ -3,15 +3,35 @@ import { InjectModel } from '@nestjs/mongoose';
 import { Types, Model } from 'mongoose';
 import { Order } from '../entities/order.entity';
 import { CreateOrderDto } from '../dto/create-order.dto';
+import {RabbitPublisherService} from '../rabbit-publisher/rabbit-publisher.service'
 
 @Injectable()
 export class OrderService {
-  constructor(@InjectModel(Order.name) private readonly orderModel: Model<Order>) { }
+  constructor(@InjectModel(Order.name) private readonly orderModel: Model<Order>,private readonly rabbitPublisherService: RabbitPublisherService) { }
 
   async create(createOrderDto: CreateOrderDto): Promise<{ order: Order; status: HttpStatus }> {
     try {
       const createdOrder = new this.orderModel(createOrderDto);
+     
       const savedOrder = await createdOrder.save();
+      const message = {
+        pattern: 'message_queue',
+        data: {
+          // to: savedOrder.user.email,
+          to:"efrat1574@gmail.com",
+          subject:"message about a new order",
+          html: "", 
+          type: 'email',
+          kindSubject: 'orderMessage',
+          numOrder:savedOrder.id,
+          nameBussniesCode:savedOrder.businessCode,
+          date:`${savedOrder.date.getUTCDate()}/${savedOrder.date.getUTCMonth()}/${savedOrder.date.getUTCFullYear()}`
+          
+        },
+      };
+console.log("d",message.data);
```

Comment:
עדיף לכתוב משהו ברור יותר כמו mail data

Reply:
זה לאהיה החלטה שלי ככה הם הגדירו את זה( interface)

Reply:
חשבתי שהתכוונת ל-to

Reply:
זה רק היה בשבילי ושחכתי למחורק את זה,

Reply:
ושכחתי